src/frontend/src/components/anchorPicker.ts

```diff
@@ -1,91 +1,43 @@
 import { html, TemplateResult } from "lit-html";
 import { arrowRight } from "./icons";
 import { mount } from "../utils/lit-html";
-import { mkAnchorInput } from "./anchorInput";
-import { until } from "lit-html/directives/until.js";
+import { NonEmptyArray } from "../utils/utils";
 
 type PickerProps = {
-  savedAnchors: bigint[];
+  savedAnchors: NonEmptyArray<bigint>;
   pick: PickCB;
-  button: string;
-} & Links;
-
-type Links = {
-  recoverAnchor: () => void;
-  register: () => void;
-  addDevice: () => void;
+  moreOptions: () => void;
 };
 
 type PickCB = (userNumber: bigint) => void;
 
-/** A component for picking an anchor number. One big list where the first
- * elements are (clickable) anchors, and the last (and sometimes only) element
- * is a menu for using any anchor or creating a new anchor. */
+/** A component for picking an anchor number. One big list where the elements are (clickable)
+ * anchors */
 export const mkAnchorPicker = (
   props: PickerProps
 ): {
   template: TemplateResult;
 } => {
-  const template = html`<ul class="c-list c-list--anchors l-stack">
-    ${elements(props)}
-  </ul>`;
-
-  return { template };
-};
-
-// The list elements, including the anchor creation menu
-function elements(props: PickerProps): TemplateResult[] {
-  const otherAnchorMenuTpl = otherAnchorMenu(props);
-  const savedAnchorsTpl = props.savedAnchors.map((anchor, i) =>
+  const elems = props.savedAnchors.map((anchor, i) =>
     anchorItem({ anchor, pick: props.pick, focus: i == 0 })
   );
 
-  let elems = [];
-
-  if (savedAnchorsTpl.length > 0) {
-    // If there are saved anchors, show those, and then an expandable menu
-    // for "other anchor".
-
-    // Function that replaces the "other anchor" button with the "other anchor" menu
-    // (actually a reference written when the template is created)
-    let otherAnchorOpen: (() => void) | undefined = undefined;
-
-    const otherAnchorOffer: TemplateResult = html`
-      <button
-        class="t-link c-list__parcel c-list__parcel--fullwidth c-list__parcel--summary"
-        @click="${() => {
-          otherAnchorOpen?.();
-        }}"
-        @focus="${() => {
-          otherAnchorOpen?.();
-        }}"
-        data-role="more-options"
-      >
-        More options<i class="c-list__icon"> … </i>
-      </button>
-    `;
+  elems.push(html` <li class="c-list__item c-list__item--noFocusStyle">
```

Comment:
nth of nth: in nns-dapp, opiniated as I am, I would probably complain about the code style 😅

```
export const mkAnchorPicker = (
  {savedAnchores, pick}: PickerProps
)
     ...
     const more = html(`<li>...</li>`);
     const elems = [
         more,
          ....savedAnchors.map(
      ]
```

I generally deconstruct everything I can and try to avoid to modify state, so no `push` and `for` if I don't have too.